lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```
@@ -0,0 +1,128 @@
+                expect(teamEditView.$('.wrapper-msg .title').text().trim()).toBe("Oops!");
+                var errorMessage = "Your team could not be created. Check the highlighted fields below and try again.";
+                expect(teamEditView.$('.wrapper-msg .copy').text().trim()).toBe(errorMessage);
+            };
+
+        beforeEach(function () {
+            setFixtures('<div class="teams-content"></div>');
+            spyOn(Backbone.history, 'navigate');
+            teamEditView = new TeamEditView({
+                el: $('.teams-content'),
+                teamParams: {
+                    topicId: 'awesomeness',
+                    topicName: 'Awesomeness',
+                    languages: [['a', 'aaa'], ['b', 'bbb']],
+                    countries: [['c', 'ccc'], ['d', 'ddd']]
+                }
+            }).render();
+
+            teamEditView.$('.wrapper-msg').hide();
+        });
+
+        it('can create a team', function () {
+            var requests = AjaxHelpers.requests(this);
+
+            teamEditView.$('.u-field-name input').val('TeamName');
+            teamEditView.$('.u-field-textarea textarea').val('TeamDescription');
+            teamEditView.$('.u-field-language select').val('a').attr("selected", "selected");
+            teamEditView.$('.u-field-country select').val('c').attr("selected", "selected");
+
+            teamEditView.$('.create-team.form-actions .action-primary').click();
+            AjaxHelpers.respondWithJson(requests, {
+                "id": "tn",
+                "name": "TeamName",
+                "is_active": true,
+                "course_id": "a/b/c",
+                "topic_id": "awesomeness",
+                "date_created": "2015-07-29T09:59:37.528Z",
+                "description": "TeamDescription",
+                "country": "c",
+                "language": "a",
+                "membership": []
+            });
+
+            expect(teamEditView.$('.wrapper-msg').is(':visible')).toBeFalsy();
```

in jasmine css is not applied so this message banner is shown by default thats why i hide it in the beforeEach. i will check it though.